app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -82,10 +75,17 @@ import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val SMALL_TEXT_SIZE = 0
-private const val MEDIUM_TEXT_SIZE = 5
-private const val LARGE_TEXT_SIZE = 10
+private const val SMALL_TEXT_SIZE_SCALE = 0.8f
+private const val MEDIUM_TEXT_SIZE_SCALE = 1.0f
+private const val LARGE_TEXT_SIZE_SCALE = 1.2f
+private const val EXTRA_LARGE_TEXT_SIZE_SCALE = 1.4f
```

Comment:
@rt4914 should we use the same variables from the main class `ReadingTextSizeFragment`?

Reply:
@anandwana001 I was thinking to keep it this way because by doing so we make sure the text sizes are exactly in the correct ratio as in the mocks. If we let these values to come from any other file then the tests will pass even for the change in these values (which is not something that we expect, as changing the text ratio for the complete app will be discussed 
thoroughly)

Reply:
Let me rephrase, the test is testing that the text size should be in the correct scale of small/medium/large/etc.
The production scaling is happening here - https://github.com/oppia/oppia-android/blob/7adefe2532359617badd83f017dff029e4a3a26d/app/src/main/java/org/oppia/android/app/options/ReadingTextSizeFragmentPresenter.kt#L144

So, our test should verify that this production scaling is happening correctly, so that's the reason I am thinking to use the const from `ReadingTextSizeFragmentPresenter` as a single source for both production and testing.